### PR TITLE
[BE] Set up heartbeat endpoint (heartbeat service) that accepts calls from FE - refactored code as per Scott comment

### PR DESCRIPTION
**Pull request by tonyxuan2021**
- refactored business logic code as per Scott comment
- added readme file for using database instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Getting Started
```
Comment (yhus) [inline README.md:1]: This is great!

File: app.py [hunk 2]
```diff
@@ -39,27 +42,39 @@ class HeartbeatDriver(db.Model):
...
 
     return "heartbeat data added successfully", 200
 
 
 if __name__ == "__main__":
 
     app.run(debug=True)
+
+
+def post_heartbeat(id, role, timestamp, lat, long, spd):
```
Comment (yhus) [inline app.py:68]: Would always prefer readability in these cases - speed is only 2 more chars but way easier to understand (esp out of context)

**Review by yhus — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Clone the repo to your local device `git clone git@github.com:BFT-Smartbus/smartbus-heartbeat.git`
+
+## Setting up Python virtual environment
+
+1. Set up a virtual environment in root folder: `python3 -m venv venv`
+2. Activate the virtual environment: `source .venv/bin/activate`
```
Comment (chunloy) [inline README.md:8]: one small change: `source .venv/bin/activate` --> `source venv/bin/activate`

**Review by chunloy — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (tonyxuan2021) [reply app.py:68]: Sure I will change to speed.

File: README.md (hunk 3 above)
Comment (tonyxuan2021) [reply README.md:8]: sure. I will change.

**Review by chunloy — APPROVED** [review]

**Pull request merged**